tests/interface/TWTransactionCompilerTests.cpp

```diff
@@ -107,7 +107,7 @@ TEST(TWTransactionCompiler, ExternalSignatureSignBinance) {
 }
 
 TEST(TWTransactionCompiler, ExternalSignatureSignEthereum) {
-    /// Step 1: Prepare transaction input (protobuf)
+    /// Step 1: Prepare transaction signingInput (protobuf)
```

Comment:
just renaming here? I don't see any problem with `input` here

Reply:
IDE renaming, will fix it